DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
-from Configuration.StandardSequences.Eras import eras
+#from Configuration.StandardSequences.Eras import eras
 
-process = cms.Process("PIXELDQMLIVE", eras.Run2_2017)
+process = cms.Process("PIXELDQMLIVE")
```

Comment:
Why remove the era?

Reply:
at the moment there no suitable era and if I keep this one I get immediately a crash

Anyway, is the era duing anything in Tracker local reco?

Reply:
> at the moment there no suitable era and if I keep this one I get immediately a crash

Ok (interesting, what kind of crash?)

> Anyway, is the era duing anything in Tracker local reco?

Yes, it enables the "phase1 mode" of RawToDigi
https://github.com/cms-sw/cmssw/blob/26f67af4327f839c41bb86555eb523cd9737ed19/EventFilter/SiPixelRawToDigi/python/SiPixelRawToDigi_cfi.py#L22
and tunes clusterizer parameters
https://github.com/cms-sw/cmssw/blob/26f67af4327f839c41bb86555eb523cd9737ed19/RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizer_cfi.py#L32-L40

Reply:
right was about to give the same, the era is needed for the phase1 pixel (even before the tracking)

Reply:
ok @boudoul and @makortel , I will add it back as soon as we have a "working era", is it used also for strip local reco?

Reply:
well, looking at the changes are so few that I could even add them manually ...